### PR TITLE
Added definition for an upcoming Pimoroni EInk board

### PR DESCRIPTION
**Pull request by ZodiusInfuser**
Everything seems to check out on my end, but let me know if you spot anything wrong. It's been a while since I last submitted a board header.

### CODE REVIEW

**Review by kilograham — DISMISSED** [review]

File: src/boards/include/boards/pimoroni_badger2040.h [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+#ifndef PICO_DEFAULT_UART_TX_PIN
+#define PICO_DEFAULT_UART_TX_PIN BADGER2040_TX_PIN
+#endif
+
+#ifndef PICO_DEFAULT_UART_RX_PIN
+#define PICO_DEFAULT_UART_RX_PIN BADGER2040_RX_PIN
+#endif
+
+// --- LED ---
+// Included so basic examples will work, and set it to the green LED
```
Comment (lurch) [inline src/boards/include/boards/pimoroni_badger2040.h:142]: https://shop.pimoroni.com/products/badger-2040 says the LED is white? :wink:

Comment (ZodiusInfuser) [reply src/boards/include/boards/pimoroni_badger2040.h]: Removed that comment now

**Comment by ZodiusInfuser** [comment]
Removed the comment @lurch pointed out.

Also, I didn't actually realise this PR hadn't been merged yet... the product's been out for a month already.

**Review by lurch — APPROVED** [review]

**Review by kilograham — APPROVED** [review]

**Comment by lurch** [comment]
Should this have been added to the 1.3.1 milestone @kilograham ?

**Pull request merged**